File: crossecore-java/src/java_/EnumConstantDeclaration.java
package java_;
public interface EnumConstantDeclaration 
 extends BodyDeclaration,VariableDeclaration
{
	
	public static com.crossecore.ocl.QuickSet<EnumConstantDeclaration> allInstances_ = new com.crossecore.ocl.QuickSet<EnumConstantDeclaration>(EnumConstantDeclaration.class);
	
		
	public static com.crossecore.ocl.QuickSet<EnumConstantDeclaration> allInstances(){
		
		com.crossecore.ocl.QuickSet<EnumConstantDeclaration> result = new com.crossecore.ocl.QuickSet<EnumConstantDeclaration>(EnumConstantDeclaration.class);
		result.addAll(EnumConstantDeclaration.allInstances_);
		
		
		return result;
	}
	
	//public static com.crossecore.ocl.QuickSet<EnumConstantDeclaration> allInstances = new com.crossecore.ocl.QuickSet<EnumConstantDeclaration>(EnumConstantDeclaration.class);
	
	
	AnonymousClassDeclaration getAnonymousClassDeclaration();
	void setAnonymousClassDeclaration(AnonymousClassDeclaration value);
	com.crossecore.ocl.OrderedSet<Expression> getArguments();
	
}

File: crossecore-typescript/java_/EnumConstantDeclaration.ts
import {OrderedSet} from "ecore/OrderedSet";
import {Set} from "ecore/Set";
import {Expression} from "./Expression";
import {AnonymousClassDeclaration} from "./AnonymousClassDeclaration";
import {BodyDeclaration} from "./BodyDeclaration";
import {VariableDeclaration} from "./VariableDeclaration";

export interface EnumConstantDeclaration
extends BodyDeclaration,
VariableDeclaration

{
	
	anonymousClassDeclaration:AnonymousClassDeclaration;
	arguments_: OrderedSet<Expression>;
	

}

